documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,267 @@
+= Cluster Controller
+
+The cluster controller is in charge to deploy a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a 
+Kafka Connect cluster which connects to an already existing Kafka cluster.
+
+When up and running, it starts to "watch" for ConfigMaps containing the Kafka or Kafka Connect cluster configuration.
+Such a ConfigMaps have to have a specific label which by default is `strimzi.io/kind=cluster` (as described later) that 
+can be changed through a corresponding environment variable.
+
+When a new ConfigMap is created on the Kubernetes/OpenShift cluster, the controller gets the deployment configuration from
```

Comment:
_deployment configuration_ might be confusing gien the Kubernetes resource called Deployment. MAybe using _cluster configuration_ would be better. You used the term cluster already before.